shdb/service.go

```diff
@@ -9493,6 +9495,43 @@ func (s *service) GetListSlugMappings() (res *models.ListSlugMappings, err error
 	return
 }
 
+func (s *service) IsProjectSkipped(project string) (isSkipped bool, err error) {
+	log.Info("IsProjectSkipped")
+	defer func() {
+		log.Info(fmt.Sprintf("IsProjectSkipped(exit): res:%+v err:%v", isSkipped, err))
+	}()
+	var ary []*models.SlugMapping
+	ary, err = s.FindSlugMappings([]string{"is_disabled", "da_name"}, []interface{}{1, project}, false, nil)
+	if err != nil {
+		isSkipped = false
+		return
+	}
+	if len(ary) == 0 {
+		isSkipped = false
+		return
+	}
+	isSkipped = true
+	return
+}
+
+func (s *service) GetSkippedProjects() (res map[string]bool, err error) {
+	res = make(map[string]bool)
+	log.Info("IsProjectSkipped")
+	defer func() {
+		log.Info(fmt.Sprintf("IsProjectSkipped(exit): res:%+v err:%v", res, err))
+	}()
+	var ary []*models.SlugMapping
+	ary, err = s.FindSlugMappings([]string{"is_disabled"}, []interface{}{1}, false, nil)
+	if err != nil {
+		return
+	}
+	for _, item := range ary {
+		item.DaName = strings.ReplaceAll(item.DaName, "/", "-")
```

Comment:
Just wonder why we replace `/` with `-` here and in `SkipDisabledProjects`?

Reply:
yes, I tested & it is not needed.